drivers/clock_control/Kconfig.mcux_mcg

```diff
@@ -0,0 +1,12 @@
+# Kconfig - MCUXpresso SDK MCG
+#
+# Copyright (c) 2019 Vestas Wind Systems A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CLOCK_CONTROL_MCUX_MCG
```

Comment:
Rather than creating a new clock control driver, can we unify this with the existing SIM clock control driver? The k64 also has the MCG. I don't remember offhand why I chose to call the driver `clock_control_mcux_sim` rather than `clock_control_mcux_mcg` (I do remember debating between the two)

Reply:
@MaureenHelm Well, the MCG and the SIM are two different modules, controlling different clocks in the SoC (e.g. the MCG Fixed Clock is enabled through the MCG, not the SIM). The KE1xF series has the SIM module, but not the MCG?

Reply:
@MaureenHelm Another option to introducing the MCG clock control driver (at least for the current use here) could be to utilise the newly introduced `fixed-clock` DTS node.